packages/connection-form/src/components/connection-form.tsx

```diff
@@ -303,10 +315,12 @@ type ConnectionFormPropsWithoutSettings = {
   darkMode?: boolean;
   initialConnectionInfo: ConnectionInfo;
   connectionErrorMessage?: string | null;
+  disableEditingConnectedConnection?: boolean;
```

Comment:
Optional new prop with undefined acting like false, so vscode should be unaffected.